### ISSUE TITLE
Проблема с scipy, etc

### ISSUE THREAD

**Issue by zvlad** [DavidRomanovizc/DatingBot#69]
1. Проблема при запуске команды python3.11 app.py
ошибка:
ImportError: The `scipy` install you are using seems to be broken, (extension modules cannot be imported), please try reinstalling.

2. В файле /DatingBot/django_project/telegrambot/telegrambot/settings.py уканан путь 
sys.path.append("D:\\Code\\DatingBot")
Это я так понимаю поменять на свой?

3. Команда python3.11 django_app.py runserver запускает бота полностью или как? Зачем еще команда python3.11 app.py ?
Почему о ней нету ниче в документации?

4. ЮРЛ для вебхуков где писать ?

5. /DatingBot/bot.conf как то под себя надо настраивать или там все верно?

6. Как запустить админку на домене? Или почему при выполении команды python3.11 django_app.py runserver мой_ип:порт и при переходе по указанопу ИП с потном ничего не отображается?

**Comment by DavidRomanovizc** [comment]
1. Попробуй эту команду `pip install --upgrade nudenet`
2. Да
3. Команда `py django_app.py runserver` запустит только Django. Этот файл был создан для того, чтобы было легче делать миграции. А команда `py app.py запускает
4. Мы не предусматривали использования вебхуков, но если очень надо, то можешь настроить [ДОКУМЕНТАЦИЯ](https://docs.aiogram.dev/en/latest/examples/webhook_example.html)
5. Если не запустится, то да, надо настраивать под себя
6. django_app.py создавался для того, чтобы упростить создание миграций, используй `py manage.py runserver 0.0.0.0:8000` и добавь "*" в allowed hosts в settings.py

**Comment by zvlad** [comment]
Выполнил все что написал, потестил с разными параметрами - не работает. Таке чуство, что некоторых файлов все таки не хватает.
Возможно, на локалке при разработке одно, а сюда залил другое.
python3.11 app.py runserver 0.0.0.0:8001
Это запускается, но при переходе по ссылке - ничего не отображается. Просто ошибка соединения.

Так же в фале /DatingBot/django_project/telegrambot/telegrambot/settings.py есть еще вот такое.
'BACKEND': 'django.template.backends.django.DjangoTemplates',
        'DIRS': [
            os.path.join(BASE_DIR, "frontend/build"),
        ],

Т.е. папки frontend/build вобще нету в проекте. django.template.backends.django.DjangoTemplates - этого тоже нету.

**Comment by DavidRomanovizc** [comment]
> Выполнил все что написал, потестил с разными параметрами - не работает. Таке чуство, что некоторых файлов все таки не хватает. Возможно, на локалке при разработке одно, а сюда залил другое. python3.11 app.py runserver 0.0.0.0:8001 Это запускается, но при переходе по ссылке - ничего не отображается. Просто ошибка соединения.
> 
> Так же в фале /DatingBot/django_project/telegrambot/telegrambot/settings.py есть еще вот такое. 'BACKEND': 'django.template.backends.django.DjangoTemplates', 'DIRS': [ os.path.join(BASE_DIR, "frontend/build"), ],
> 
> Т.е. папки frontend/build вобще нету в проекте. django.template.backends.django.DjangoTemplates - этого тоже нету.

Django используется для админ панели(`http://127.0.0.1:8000/admin`) и для orm. Изначально была идея сделать фронтенд на реакте, он уже был настроен, но я передумал. Если нужен настроенный реакт, то посмотри в истории коммитов +- 30 октября 2022 года, но там уже код устаревший

**Comment by zvlad** [comment]
Но, по идеи и самый последний коммит должен работать, так? Т.е. я запускаю по команде 
`python3.11 django_app.py runserver 0.0.0.0:8001`
и админка должна быть доступна по адресу `http://0.0.0.0:8001/admin`
и работать Телеграм-бот, от которого АПИ ключ установлен, да ?

**Comment by DavidRomanovizc** [comment]
> Но, по идеи и самый последний коммит должен работать, так? Т.е. я запускаю по команде `python3.11 django_app.py runserver 0.0.0.0:8001` и админка должна быть доступна по адресу `http://0.0.0.0:8001/admin` и работать Телеграм-бот, от которого АПИ ключ установлен, да ?

Он работает. У тебя Django не запускается, потому что url неправильный. У нас в ALLOWED_HOSTS пусто, если тебе нужен 0.0.0.0, то добавь туда звездочку 

```py
# settings.py
ALLOWED_HOSTS = ["*"]
```

И нужно использовать 8000 порт, а не 8001. А телеграм бот запускается в файле `app.py` и все. У нас Django только для админ панели и базы, а api у нас не реализован

**Comment by zvlad** [comment]
А почему 8000 порт? Если он занят уже? На 8000 порту сайт находится. Я хочу на 8001 запустить. Порт то же указывается не просто так.
ALLOWED_HOSTS = ["*"] это сделал.

**Comment by DavidRomanovizc** [comment]
> А почему 8000 порт? Если он занят уже? На 8000 порту сайт находится. Я хочу на 8001 запустить. Порт то же указывается не просто так. ALLOWED_HOSTS = ["*"] это сделал.

Так, сейчас сам проверил, с ALLOWED_HOSTS = ["*"] 8001 порт работает, но нужно явно это указать 

`python django_app.py runserver 0.0.0.0:8001`

**Comment by zvlad** [comment]
Оно то запускается, но по адресу ничего не выводится. Просто не доступен "сайт". Хотя бы белый екран быть. Ощибка соеденения.
Мне кажется, что каких то файлов не хватает. Или какие то жеские привязки к версий библиотек. Т.е. на новом оборудование ниче не работает.
